CMakeLists.txt

```diff
@@ -318,7 +318,16 @@ else()
     usFunctionCheckCompilerFlags(-Wno-mismatched-tags US_CXX_FLAGS)
   endif()
 
-  usFunctionCheckCompilerFlags("-fvisibility=hidden -fvisibility-inlines-hidden" _have_visibility)
+  # Issue #56: Workaround a clang compiler issue (https://llvm.org/bugs/show_bug.cgi?id=26749).
+  # The typeinfo symbols do not have reliable symbol visibility. Instead of exporting all
```

Comment:
You are probably referring to the previous workaround, right? But without that knowledge, I think the comment is confusing when looking at the else branch below.